node/network/bridge/src/lib.rs

```diff
@@ -34,7 +34,7 @@ use polkadot_node_network_protocol::{
 
 /// Peer set info for network initialization.
 ///
-/// To be added to [`NetworkConfiguration::extra_sets`].
+/// To be added to [`FullNetworkConfiguration`]().
```

Comment:
```suggestion
/// To be passed to [`FullNetworkConfiguration::add_notification_protocol`](sc_network::config::FullNetworkConfiguration::add_notification_protocol).
```

Reply:
I've applied your suggestions. I'm still leaving the () link part at the end empty though; rustdoc doesn't seem to be able to resolve `sc_network::config::FullNetworkConfiguration::add_notification_protocol`, and that's triggering the aforementioned crashing bug in 1.70.

Reply:
Then this should be reported upstream.